frontend/src/sass/_all.scss

```diff
@@ -21,6 +21,7 @@
 @import 'elements/figure';
 // Components -------------- //
 @import 'components/breadcrumbs';
+@import 'components/feedback';
```

Comment:
`feedback` isn't really a UI component. It's an area in the application that makes use of various UI components. nothing to change here though, as this follows the existing pattern..